crossbar/worker/router.py

```diff
@@ -689,12 +689,15 @@ def _create_resource(reactor, path_config, templates, log, cbdir, _router_sessio
 
         path_options = path_config.get('options', {})
 
+        auth = path_config.get('auth')
+
```

Comment:
Does path_config always have an 'auth' key? Should this be `.get('auth', None)`..?